engine/src/mbliphonesound.mm

```diff
@@ -306,9 +309,6 @@ bool MCSystemPlaySound(MCStringRef p_sound, bool p_looping)
 	
 	MCValueRelease(s_sound_file);
```

Comment:
I recommend either changing `s_sound_file` to an `MCAutoStringRef` or, if that is too much refactoring, making sure that it is set to `nullptr` after its contents are released.